opossum-tool/OpossumUI
# feat: handle saving of first-party attributions

### Summary of changes

- when saving a first-party attribution, delete third-party attributes
- simplify stripping functions

### Context and reason for change

Closes #2347.

### How can the changes be tested

Modify a third-party attribution to become first-party and notice that the third-party attributes are removed after saving. That is, if after saving you switch back to third-party, you won't see the deleted values any longer. Previously, a save operation would not remove these values.
